django/test/client.py

```diff
@@ -292,11 +298,12 @@ def _get_path(self, parsed):
         # If there are parameters, add them
         if parsed[3]:
             path += str(";") + force_str(parsed[3])
-        path = uri_to_iri(path).encode(UTF_8)
+        path = unquote_to_bytes(path) if six.PY3 else unquote(path)
+        path = repercent_broken_unicode(path)
         # Under Python 3, non-ASCII values in the WSGI environ are arbitrarily
         # decoded with ISO-8859-1. We replicate this behavior here.
         # Refs comment in `get_bytes_from_wsgi()`.
-        return path.decode(ISO_8859_1) if six.PY3 else path
+        return path.decode('iso-8859-1') if six.PY3 else path
```

Comment:
Not sure, I haven't looked into the history of the constants but I figured they were to prevent typos in the strings.